src/utils/conversion.js

```diff
@@ -84,6 +84,18 @@ const importProperties = (exportedFmt, initialProperties) => {
         case 'qStateName':
           newProperties[key] = exportedFmt.properties[key];
           break;
+        case 'showTitles':
+          newProperties[key] = exportedFmt.properties[key];
+          break;
+        case 'title':
+          newProperties[key] = exportedFmt.properties[key];
+          break;
+        case 'subtitle':
+          newProperties[key] = exportedFmt.properties[key];
+          break;
+        case 'footnote':
+          newProperties[key] = exportedFmt.properties[key];
+          break;
```

Comment:
When using the same code you only need to declare the case statements. Like this:

```
case 'a':
 case 'b':
    do something;
    break;
```